### ISSUE TITLE
Links to section within a journal entry page not working.

### ISSUE THREAD

**Issue by farling42** [HavlockV/CoC7-FoundryVTT#1274]
If I include a link to a section within a journal page, such as 
```
@UUID[JournalEntry.Qdr4gy63qpopczui.JournalEntryPage.hZlVZlCWMToeRmPO#another-header]{Another Header}
```

Clicking on the link only opens the specific page within the journal, it does not honour the section header `"#another-header"`

This is working properly in other game systems, but the "data-hash" parameter that is put onto the "a href" in the HTML of the journal page containing the button does not seem to be getting passed to the journal opening routine.

**Comment by farling42** [comment]
With some debugging, it looks like the originalTarget is wrong in the event passed to TextEditor._onClickContentLink

In CoC7 the originalTarget is the entire window, whereas in other systems it is just the `a.content-link` element

Tracing _onClickContentLink further, the event has the correct currentTarget on entry, but after "await fromUuid" the event.currentTarget has changed to point to the window instead of the link element.

**Comment by snap01** [comment]
A fix for this should be included in the next release

**Comment by snap01** [comment]
Can you please confirm if this issue can be closed with the latest 0.9.2 release?

**Comment by farling42** [comment]
Yes, this is working correctly in 0.9.2
Thanks very much for providing the fix.
Users of "PDF Pager" will be very happy :-)
